### ISSUE TITLE
Mathematical Framework of Bias-Variance Trade-off and Backpropagation

### ISSUE THREAD

**Issue by vaibhawvipul** [PyDataPune/Talks#53]
Usually we see Bias-Variance trade off as a concept which we memorise. However the beauty of Bias-Variance is profound once we see how beautifully it appears in the equations of the loss functions. In this talk we will be seeing the mathematical occurrence of bias-variance tradeoff in ML.

The second part of the talk will lay a mathematical framework of Back-Propagation. This talk will be based on Yann LeCunn's paper. [Link to the paper](http://yann.lecun.com/exdb/publis/pdf/lecun-88.pdf)
